services/harmony/app/middleware/shapefile-converter.ts

```diff
@@ -83,6 +91,82 @@ const contentTypesToConverters = {
   'application/shapefile+zip': { name: 'ESRI Shapefile', geoJsonConverter: _esriToGeoJson },
 };
 
+/**
+ * Compute the number of sides for the Polygon that will approximate a circle of the given
+ * radius
+ * @param radius - radius of the circle in meters that is centered on a Point
+ * @returns the radius
+ */
+export function numberOfSidesForPointCircle(radius: number): number {
+  return Math.ceil((env.maxPointCircleSides - 3) * (1.0 - env.pointCircleFunctionBase ** radius) + 3);
+}
+
+/**
+ * Compute a polygon approximating the circle centered at the given point with the given radius
+ * or a default radius if none is provided
+ * NOTE: Radius is read from the the `properties` field of the `Point` and should be in meters.
+ * The `properties` field is not preserved
+ *
+ * @param pnt - geojson Point for the center of the circle
+ * @returns a geojson polygon
+ */
+function pointToPolygon(pnt: Feature<Point>): Feature<Polygon, {}> {
```

Comment:
Just FYI that we'll want to do something similar for the EDR radius function in the future - https://opengeospatial.github.io/e-learning/ogcapi-edr/text/radiusquery.html